substrate/src/lib.rs

```diff
@@ -100,10 +113,9 @@
 //! * `sp-*` (short for *substrate-primitives*) crates, located under `./primitives` folder. These
 //!   are the traits that glue the client and runtime together, but are not opinionated about what
 //!   framework is using for building the runtime. Notable examples are [`sp-api`] and [`sp-io`],
-//!   which form the communication bridge between the client and runtime, as explained in
-//!   [`substrate_diagram`].
+//!   which form the communication bridge between the client and runtime.
 //! * `pallet-*` and `frame-*` crates, located under `./frame` folder. These are the crates related
-//!   to FRAME. See [`frame_support`] for more information.
+//!   to FRAME. See [`frame`] for more information.
```

Comment:
Line 132 typo `commandsOnce`.

Reply:
I moved all doc related comments to https://github.com/paritytech/polkadot-sdk/compare/kiz-frame-api...kiz-umbrella-crates?expand=1, will be reflected there.